PhysicsTools/PatAlgos/plugins/GenJetFlavourInfoPreserver.cc

```diff
@@ -78,15 +74,21 @@ pat::GenJetFlavourInfoPreserver::produce(edm::Event & iEvent, const edm::EventSe
 
 	
 
-    for (View<reco::GenJet>::const_iterator it = GenJets->begin(), ed = GenJets->end(); it != ed; ++it) {
+    for (View<reco::GenJet>::const_iterator it = genJets->begin(), ed = genJets->end(); it != ed; ++it) {
         if (!cut_(*it)) continue;
```

Comment:
The part related to this loop and synchronization with the slimmedJets still needs a rewrite

The following comments were not addressed:
https://github.com/cms-sw/cmssw/pull/19048#discussion_r120653178
https://github.com/cms-sw/cmssw/pull/19048#discussion_r120656241